packages/web-app-files/src/composables/keyboardActions/useKeyboardActionsSearchTable.ts

```diff
@@ -54,9 +55,11 @@ export const useKeyboardActionsSearchTable = (keyActions, paginatedResources) =>
       return
     }
     resetSelectionCursor()
-    await store.dispatch('Files/resetFileSelection')
+    store.dispatch('Files/resetFileSelection')
+    await nextTick()
     store.commit('Files/ADD_FILE_SELECTION', { id: nextId })
-    scrollToResource({ id: nextId } as any)
+    await nextTick()
```

Comment:
Not sure if we should await the nextTick within `scrollToResource` instead of spamming it outside though

Reply:
What do you mean?